README.md

```diff
@@ -1,2 +1,363 @@
-# learn-payment-processing
-[WiP] Learn how to process online payments in your web application! :money:
+<div align="center">
+
+# Learn Payment Processing
+
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/dwyl/learn-payment-processing/ci.yml?label=build&style=flat-square&branch=main)
+[![codecov.io](https://img.shields.io/codecov/c/github/dwyl/learn-payment-processing/main.svg?style=flat-square)](https://codecov.io/github/dwyl/phoenix-chat-example?branch=main)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat-square)](https://github.com/dwyl/phoenix-chat-example/issues)  
+[![HitCount](https://hits.dwyl.com/dwyl/learn-payment-processing.svg?style=flat-square&show=unique)](http://hits.dwyl.com/dwyl/learn-payment-processing)
+  
+Learn what payment processing is 
+and how you can add it to your application!
+
+</div>
+
+# Why? 🤷
+
+Not all applications are free.
+There are some that can be acquired through
+a [one time purchase](https://git-fork.com/buy).
+There are others
+that are [subscription-based](https://www.notion.so/pricing).
```

Comment:
Almost _all_ recent **`SaaS`** is subscription based.
We could have a _list_ of all the well-known **`SaaS`** products in a _footnote_.

we're _definitely_ not going to link to [`Notion`](https://user-images.githubusercontent.com/194400/85288902-75077480-b48e-11ea-8b41-75299936985b.png) in the intro paragraph to our payment processing tutorial. 🤣 
It's like FCP linking to [FC Barcelona](https://www.fcbarcelona.com/en/) on their home page ... 😜 
Not saying we will "compete" with `Notion`, but there will be some "overlap" and the _last_ thing we want is to send them _any_ more traffic. 👌

Reply:
you're right
fixed on [`52e8e50` (#2)](https://github.com/dwyl/learn-payment-processing/pull/2/commits/52e8e50ea1dda267ef96ff6f8bd68941c1e1fce4)